frontend/pages/SoftwarePage/components/SoftwareDetailsSummary/_styles.scss

```diff
@@ -32,6 +32,9 @@
 
   &__data-set {
     font-size: $x-small;
+    display: flex;
+    flex-direction: column;
+    gap: $pad-xsmall;
```

Comment:
I'm breaking this out into its own component, so just heads up that we'll be removing this style altogether soon

Reply:
Oh sweet. Hopefully there's time to iterate this sprint.

Reply:
Already done in my branch, so we'll just have to reconcile it, shouldn't be too hard